package_drupal_script.sh

```diff
@@ -15,8 +15,12 @@ STAGING_DIR_NAME=drupal_sprint_package
 STAGING_DIR_BASE=~/tmp
 STAGING_DIR=$STAGING_DIR_BASE/$STAGING_DIR_NAME
 REPO_DIR=$PWD
+QUICKSPRINT_RELEASE=v0.0.6
```

Comment:
Please use `QUICKSPRINT_RELEASE=$(git describe --tags --always --dirty)` instead of wiring to something that has to be changed all the time.

Reply:
done